Description: Provide version#"git description".
 Since we may not be building the package from a git repository, we
 hardcode the git description ahead of time.  This patch is
 automatically updated by debian/scripts/m2-get-orig-source.sh.
Author: Doug Torrance <dtorrance@debian.org>
Forwarded: not-needed
Last-Update: 2020-12-21

--- a/M2/configure.ac
+++ b/M2/configure.ac
@@ -506,10 +506,8 @@
     *) AC_MSG_ERROR(unrecognized option for enable-compress) ;;
 esac
 
-AC_SUBST([GIT_DESCRIPTION],
-         [`$GIT describe --tags --dirty || echo release-$PACKAGE_VERSION`])
-AC_SUBST([GIT_BRANCH],
-         [`$GIT branch --show-current`])
+AC_SUBST([GIT_DESCRIPTION], [release-1.24.05-834-g14766aefd0])
+AC_SUBST([GIT_BRANCH], [development])
 
 AC_SUBST(DEB,no)
 AC_ARG_ENABLE(deb,  AS_HELP_STRING(--enable-deb,prepare a *.deb package (for debian, ubuntu, ...)), DEB=$enableval)
